### ISSUE TITLE
Regression since version `3.7.0`, `no-conflicting-classes` - Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by virtuallyunknown** [schoero/eslint-plugin-better-tailwindcss#203]
Greetings!

I've updated the plugin from version `3.6.3` to `3.7.3` yesterday and an error started to appear in the VSCode ESLint output console. I was able to determine that the error starts at version `3.7.0`, earlier versions work fine.

```
2025-08-04 04:36:11.124 [error] An unexpected error occurred:
2025-08-04 04:36:11.124 [error] TypeError: Cannot convert undefined or null to object
Occurred while linting /home/user/projects/better-tw-error/src/App.tsx:3
Rule: "better-tailwindcss/no-conflicting-classes"
    at Object.keys (<anonymous>)
    at lintLiterals (file:///home/user/projects/better-tw-error/node_modules/.pnpm/eslint-plugin-better-tailwindcss@3.7.3_eslint@9.32.0_jiti@2.5.1__tailwindcss@3.4.17/node_modules/eslint-plugin-better-tailwindcss/lib/esm/rules/no-conflicting-classes.js:50:20)
    at JSXOpeningElement (file:///home/user/projects/better-tw-error/node_modules/.pnpm/eslint-plugin-better-tailwindcss@3.7.3_eslint@9.32.0_jiti@2.5.1__tailwindcss@3.4.17/node_modules/eslint-plugin-better-tailwindcss/lib/esm/utils/rule.js:40:17)
    at ruleErrorHandler (/home/user/projects/better-tw-error/node_modules/.pnpm/eslint@9.32.0_jiti@2.5.1/node_modules/eslint/lib/linter/linter.js:1174:33)
    at /home/user/projects/better-tw-error/node_modules/.pnpm/eslint@9.32.0_jiti@2.5.1/node_modules/eslint/lib/linter/source-code-visitor.js:76:46
    at Array.forEach (<anonymous>)
    at SourceCodeVisitor.callSync (/home/user/projects/better-tw-error/node_modules/.pnpm/eslint@9.32.0_jiti@2.5.1/node_modules/eslint/lib/linter/source-code-visitor.js:76:30)
    at /home/user/projects/better-tw-error/node_modules/.pnpm/eslint@9.32.0_jiti@2.5.1/node_modules/eslint/lib/linter/source-code-traverser.js:291:18
    at Array.forEach (<anonymous>)
    at SourceCodeTraverser.traverseSync (/home/user/projects/better-tw-error/node_modules/.pnpm/eslint@9.32.0_jiti@2.5.1/node_modules/eslint/lib/linter/source-code-traverser.js:290:10)

```

Minimal reproducible repository can be found here - https://github.com/virtuallyunknown/better-tw-error

```tsx
// src/App.tsx
function App() {
    return (
        <h1 className="underline text-3xl font-bold invalid-class">
            Hello world!
        </h1>
    )
}
```

Here we have `invalid-class`, but it's not being detected because the VSCode eslint plugin breaks/throws. If downgrade the plugin:

- `pnpm add eslint-plugin-better-tailwindcss@3.6.3 -D`

and then restart the eslint server from the command palette:

- `> ESLint: Restart ESLint server`

the problem goes away and `invalid-class` is being correctly flagged as an error.

I should also clarify that the example above has less to do with the `no-conflicting-classes` rule itself or any other rule, and more with the fact that the whole plugin is breaking.

EDIT: I forgot to add that disabling the rule from the `eslint.config.js` file also temporarily fixes the problem. I am not really sure if that information is useful, but leaving it here just in case.

```js

export default [
    // ...
    {
        // ...
        rules: {
            ...eslintPluginBetterTailwindcss.configs["recommended-warn"].rules,
            ...eslintPluginBetterTailwindcss.configs["recommended-error"].rules,
            "better-tailwindcss/enforce-consistent-line-wrapping": ["warn", { printWidth: 100 }],
            "better-tailwindcss/no-conflicting-classes": ["off"]
            //                                           ^^^^^^^
        },
    }
];
```
